addon/plugins/table/nodes/table.ts

```diff
@@ -194,8 +215,26 @@ export function tableNodes(options: TableNodeOptions): TableNodes {
           },
         },
       ],
+      serialize(node, state) {
+        const pos = getPos(node, state.doc) as ResolvedPos;
+        // table rows are 1-indexed
+        const isEven = (pos.index() + 1) % 2 === 0;
```

Comment:
Could be simplified to `pos.index() % 2 === 1`

Reply:
very true haha :)